packages/paste-website/src/components/tokens-list/index.tsx

```diff
@@ -28,8 +27,9 @@ const Content: React.FC = (props) => <Box as="div" maxWidth="size70" minWidth="0
 export const TokensList: React.FC<TokensListProps> = (props) => {
   const {theme} = useDarkModeContext();
   const [filterString, setFilterString] = React.useState('');
-  const [tokens, setTokens] = React.useState(Tokens.tokens);
-  const [tokenCategories, setTokenCategories] = React.useState(Object.keys(tokens) as unknown as [keyof typeof tokens]);
+  const [tokens, setTokens] = React.useState<{[key: string]: Token[]}>(Tokens.tokens);
+  const tokenCategories = Object.keys(Tokens.tokens);
+  const [useJavascriptNames, setUseJavascriptNames] = React.useState(false);
 
   // The rendered tokens should update every time the filterString, props, or theme changes
   React.useEffect(() => {
```

Comment:
thought: the next line (line 36) is commented out. It doesn't seem to be informative, and I vote to remove it.

Reply:
I'd rather leave this context here for now since it's not going into `main` and it might be useful for whoever does the filter ticket. but we should remove the comments before merging in the feature branch